### PR TITLE
Feature/seed commands

### PR DESCRIPTION
**Pull request by GErpeldinger**
Hello,

I needed the commands for the seeding, I made a quick thing inspired by the others, I let you have a look.
I also added aliases for commands, it's good for slacker like me

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 vendor/
+.idea/
+.DS_Store
```
Comment (DjLeChuck) [inline .gitignore:3]: This two line should not be in the projet `.gitignore`, but in a global one: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#create-a-global-gitignore

**Review by DjLeChuck — CHANGES_REQUESTED** [review]
Sorry for the delay!

Just revert the change on the `.gitignore` and I'm ready to merge.  👍🏻

File: .gitignore (hunk 1 above)
Comment (GErpeldinger) [reply .gitignore]: Thanks for the link, don't jnow a global gitignore can be created.

**Comment by GErpeldinger** [comment]
Hello, no problem !

Change it's done ;)

**Review by DjLeChuck — APPROVED** [review]

**Comment by DjLeChuck** [comment]
Here we go with the[v2.0.3](https://github.com/umanit/phinx-bundle/releases/tag/2.0.3) 👍🏻 
Thank you for your work on the bundle!

**Pull request merged**